core/templates/dev/head/components/hint_editor_directive.html

```diff
@@ -51,4 +51,12 @@
       <div style="clear: both;"></div>
     </div>
   </div>
+
+  <div ng-if="stateHintsService.savedMemento[getIndexPlusOne() - 1] && !stateHintsService.savedMemento[getIndexPlusOne() - 1].isEmpty()">
+    <audio-translations-editor component-name="hint"
```

Comment:
Hmmm... I think, can you declare

    oppia.constant('COMPONENT_NAME_HINT', 'hint')

etc., import those, use those here and change the @ to &? Put a comment above those constants saying "do not modify these, for backwards-compatibility reasons". This will make it less likely that devs alter these in the future.

Reply:
done